src/Core/Utilities/StaticStore.cs

```diff
@@ -160,42 +157,13 @@ static StaticStore()
     /// </returns>
     public static bool IsAddonSubscriptionItem(string stripePlanId)
     {
-        if (Plans.Select(p => p.PasswordManager.StripeStoragePlanId).Contains(stripePlanId))
-        {
-            return true;
-        }
-
-        if (Plans.Select(p => p.SecretsManager?.StripeServiceAccountPlanId).Contains(stripePlanId))
+        if (Plans.Any(p =>
+                p.PasswordManager.StripeStoragePlanId == stripePlanId ||
+                (p.SecretsManager?.StripeServiceAccountPlanId == stripePlanId)))
         {
             return true;
         }
 
         return false;
```

Comment:
This is a good change. Now you can just `return Plans.Any(...)` directly, instead of using the `if` statement.